wordpress-mobile/WordPress-iOS
# P2: Add branding to Reader > Settings > Followed Sites

Fixes #15473 

This PR fixes the bug found in #15473 and updates the default site icons for P2 sites.

### To test
1. Navigate to Reader tab > Settings (gear icon) > Followed sites.
2. Scroll for a bit.
3. Notice that site icons that contain transparencies are no longer tinged gray, but instead let the background color come through.

### Screenshots

| Bug | Fix in light mode | fix in dark mode |
| --- | --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-07 at 16 15 38](https://user-images.githubusercontent.com/1062444/101922067-4e034300-3b93-11eb-895c-5df94375353a.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-11 at 08 59 15](https://user-images.githubusercontent.com/1062444/101922094-55c2e780-3b93-11eb-8969-8f0dade7a332.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-12-11 at 08 59 43](https://user-images.githubusercontent.com/1062444/101922111-58bdd800-3b93-11eb-92ee-006348786eae.png) |


PR submission checklist:
